slashing-protection/src/test/java/tech/pegasys/web3signer/slashingprotection/dao/LowWatermarkDaoTest.java

```diff
@@ -158,6 +158,69 @@ public void ensureBothWatermarksMustBeNonNull() {
         .isInstanceOf(UnableToExecuteStatementException.class);
   }
 
+  @Test
+  public void cannotUpdateAttestationWatermarkWithLowerValue() {
+    Optional<SigningWatermark> watermark;
+    insertValidator(Bytes.of(100), 1);
+
+    lowWatermarkDao.updateEpochWatermarksFor(handle, 1, UInt64.valueOf(2), UInt64.valueOf(3));
+    watermark = lowWatermarkDao.findLowWatermarkForValidator(handle, 1);
+    assertThat(watermark).isNotEmpty();
+    assertThat(watermark.get().getSourceEpoch()).isEqualTo(UInt64.valueOf(2));
+    assertThat(watermark.get().getTargetEpoch()).isEqualTo(UInt64.valueOf(3));
+
+    lowWatermarkDao.updateEpochWatermarksFor(handle, 1, UInt64.valueOf(1), UInt64.valueOf(2));
```

Comment:
If the target was greater existing target watermark but the source was lower than the existing source watermark what should happen? Think from the current logic nothing would be updated?